Tests/UnitTests/ConnectLibraryTests/ConnectMocksTests/ConnectMocksTests.swift

```diff
@@ -0,0 +1,186 @@
+// Copyright 2022-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@testable import Connect
+import ConnectMocks
+import SwiftProtobuf
+import XCTest
+
+/// Test suite that validates the behavior of generated mock classes.
+@available(iOS 13, *)
+final class ConnectMocksTests: XCTestCase {
```

Comment:
This isn't a new file but git tracked it as one since it required updates to use different generated classes